api/python/provisioner/commands/set_swupgrade_repo.py

```diff
@@ -121,11 +188,17 @@ def dynamic_validation(self, params: inputs.SWUpgradeRepo, targets: str):  # noq
 
         try:
             logger.debug("Configuring upgrade candidate repo for validation")
-            self._prepare_repo_for_apply(candidate_repo, enabled=False)
 
-            self._apply(candidate_repo, targets)
+            self._prepare_single_iso_for_apply(candidate_repo)
```

Comment:
@dmitrydb Could you add some logger statements against these calls to inform what we are attempting?

Reply:
Added logging string to `self._prepare_single_iso_for_apply` and before the first `self._apply`